spec/mailers/participant_validation_mailer_spec.rb

```diff
@@ -131,6 +131,21 @@
     end
   end
 
+  describe "#we_need_information_for_your_programme_email" do
+    let(:induction_coordinator_email) do
+      described_class.we_need_information_for_your_programme_email(
+        recipient: recipient,
+        school_name: school_name,
+        participant_start: "example.com/start-validation",
+      )
```

Comment:
This needs to be `start_url` rather than `participant_start`.

Reply:
Cheers